content/docs/studio/user-guide/account-management.md

```diff
@@ -5,24 +5,59 @@ Iterative Studio. In the menu that comes up, click on `Profile`. Your user
 profile page will open. This page has multiple sections that are described
 below.
 
+## Profile
+
+Here, you can see your name and profile picture. If you signed up with a
+GitHub.com, GitLab.com or Bitbucket.org account, these details are fetched from
```

Comment:
OK all the .coms and .orgs start to get repetitive after a bit. I'd ✂️ them (minor).

Reply:
See previous comment for why I added them.